library/src/main/java/com/bumptech/glide/load/resource/gif/GifDrawable.java

```diff
@@ -309,7 +309,23 @@ public void setLoopCount(int loopCount) {
         }
 
         if (loopCount == LOOP_INTRINSIC) {
-            maxLoopCount = decoder.getLoopCount();
+            int netscapeIterationCount = decoder.getNetscapeIterationCount();
```

Comment:
This is still in SNAPSHOT so it's possible to move it to `getNetscapeIterationCount` without breaking, what do you think? @sjudd @toyama-sumio